server/enhancedDate.js

```diff
@@ -1 +1,96 @@
-// export this as a Node module using the enhancedDate.js code from Module_2
\ No newline at end of file
+// export this as a Node module using the enhancedDate.js code from Module_2
+
+var enhancedDate = function() {
+
+    'use strict';
+    
+    var myDate = null;
+    var dateSet = false;
+    var dayNames = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturaday'];
```

Comment:
Spelling error for 'Saturday' - since it's actually Saturday when I'm testing your project, of course it didn't return anything.